todo_app/app.py

```diff
@@ -1,6 +1,6 @@
 from crypt import methods
```

Comment:
I know you removed this line on your machine, but it looks like it wasn't part of the committed git changes, but I'll assume that will come through in M3!